sources/packages/web/src/views/aest/institution/InstitutionDetails.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <h5 class="text-muted">
+    <a @click="goBack()">
+      <v-icon left> mdi-arrow-left </v-icon> Back to Institution</a
+    >
+  </h5>
+  <full-page-container>
+    <h2 color="primary-color" class="mb-15">
```

Comment:
better to add a class(Figma will have the style details) instead of heading tags